src/components/grid-aware/VideoHeader/VideoHeader.module.css

```
@@ -0,0 +1,145 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background-image: linear-gradient(90deg, #000 0%, rgba(0, 0, 0, 0) 100%), var(--background-image);
+  background-size: cover;
+  grid-column: 1/ span 3;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    display: grid;
+    grid-template-columns: 40px 1fr 40px;
+    grid-template-rows: 214px auto;
+    max-width: var(--grid-bleed-max-width);
+    padding-bottom: 55.5px;
+  }
+
+  .bleedBackground {
+    background-image: linear-gradient(360deg, #000 0%, rgba(0, 0, 0, 0) 100%), var(--background-image);
+    grid-row: 1;
+  }
+}
+
+/* Main grid */
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 150px 0;
+}
+
+.gridTextArea {
+  align-content: center;
+  grid-column: 1 / span 6;
+  grid-row: 1;
+}
+
+.gridPlayButton {
+  align-self: center;
+  grid-column: 9 / span 2;
+  grid-row: 1;
+  text-align: center;
+}
+
+@media (--tablet-and-down) {
+  .bleedMainContent {
+    grid-row: 1 /span 2;
```

Nit: Please put a space after the `/`.

```suggestion
    grid-row: 1 / span 2;
```